examples/dataflow-hashpipeline/infrastructure/hashpipeline-trigger/main.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2020 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+from google.cloud import pubsub_v1
+from os import getenv
```

Comment:
Small style nit. Import modules not functions- there are other instances of this in other files but just to point it out. http://google.github.io/styleguide/pyguide.html#22-imports